winterfell/webapp/src/js/profileCtrl.js

```diff
@@ -1,54 +1,48 @@
 'use strict';
 var app = angular.module('vetafiApp');
-app.controller('profileCtrl', ['$scope', '$location', '$window', 'Profile', 'claimService', 'net', 'strings', '$uibModal', '$state',
-  function($scope, $location, $window, Profile, claimService, net, strings, $uibModal, $state) {
-    $scope.ranks = strings.ranks;
-    $scope.branches = strings.branches;
-    $scope.insigniaUrls = {
-      airforce: '../icons/us_insignia_airforce.svg',
-      army: '../icons/us_insignia_army.svg',
-      coastguard: '../icons/us_insignia_coastguard.svg',
-      marine: '../icons/us_insignia_marine.svg',
-      navy: '../icons/us_insignia_navy.svg'
-    };
+app.controller('profileCtrl', ['$scope', '$location', '$window', 'Profile', 'claimService', 'net', '$uibModal', '$state',
+  function($scope, $location, $window, Profile, claimService, net, $uibModal, $state) {
 
-    var tabValues = {
-      military: 'military',
-      fileClaims: 'claims',
-      settings: 'settings'
-    };
-    $scope.currentTab = tabValues.military;
-    $scope.userInfo = {};
-    $scope.claims = [];
-    $scope.militaryInfo = [
-      {
-        rank: 'lieutenant',
-        branch: 'navy',
-        yearStart: '2013',
-        yearEnd: '2015'
-      },
-      {
-        rank: 'captain',
-        branch: 'army',
-        yearStart: '2012',
-        yearEnd: '2013'
-      }
-    ];
+    $scope.user = Profile.user.user;
+    $scope.claims = []; // list of user's claims
```

Comment:
You can resolve claims through the UI router and then inject them into the controller, similar to how we do for other things like the user and forms, etc.